src/containers/CallIn/CallIn.js

```diff
@@ -252,6 +252,17 @@ class CallIn extends Component {
                         </Collapse>
                         </Col></Row>
                     </section>
+
+                    {/* This covid section can be removed when the US crisis has settled. */}
+                    <section id="covid">
+                        <Row type="flex" justify="center">
+                            <Col xs={24} md={20} lg={18} xl={12}>
+                            <Alert showIcon style={{ marginBottom: 20}} type="warning" message="Calling During the COVID-19 Crisis" description="Many parts of the country are currently overwhelmed by this public health crisis. Please use your best judgement to decide whether the Member of Congress that represents your part of the country is receptive to our message during this sensitive time." />
```

Comment:
Just a tiny nit, but you can multi line this if you'd like: 

```
<Alert
    type="warning"
    description={`
        Many parts of the country are currently overwhelmed by this public health crisis. 
        Please use your best judgement to decide whether the Member of Congress that 
        represents your part of the country is receptive to our message during this sensitive time.
    `}
    ...
/>
```

Reply:
that will certainly help for readability. Will adopt this in the future. Thanks!